test/e2e/testing-manifests/storage-csi/hostpath/hostpath/csi-hostpathplugin.yaml

```diff
@@ -19,7 +19,7 @@ spec:
           args:
             - --v=5
             - --csi-address=/csi/csi.sock
-            - --kubelet-registration-path=/var/lib/kubelet/plugins/csi-hostpath/csi.sock
+            - --kubelet-registration-path=/registration/csi-hostpath/csi.sock
```

Comment:
Ditto: undo this change. It is for the driver socket not the registration socket.